docs/getting-started/installing-service-binding.md

```diff
@@ -4,13 +4,46 @@ sidebar_position: 1
 
 # Installing Service Binding
 
-## Compatiblity Note
+The Service Binding Operator can be installed on the following version of Kubernetes and OpenShift.
 
-## Operator
+- Kubernetes version 1.20 or above.
+- OpenShift version 4.7 or above.
 
-_How can To be provided soon_
+## Installing Operator using OLM
 
-## Helm
+You can install ServiceBinding from
+[OperatorHub.io](https://operatorhub.io/operator/service-binding-operator).  To
+see the installation steps, click on the blue _Install_ button.
 
-_To be provided soon_
+## Installing Operator without OLM
 
+If you don't have OLM, you can install the operator using the released configuration:
```

Comment:
not sure if we should use the term "released configuration". IMHO, these released resources.

Reply:
Done. f2c70cce6c5ef069a1e10dea2e584fed54cd294b